### PR TITLE
feat(connector): add realease note + page on connectors

### PR DESCRIPTION
**Pull request by baptistemesta**
Describe how connectors are executed using a sequence diagram

Relates to [BR-5](https://bonitasoft.atlassian.net/browse/BR-5)

### CODE REVIEW

File: md/release-notes.md [hunk 1]
```diff
@@ -122,3 +122,10 @@ A work execution audit mechanism has been introduced. It can be activated
 to detect when a work takes too much time to be executed or it was _rescheduled_
 too much times.
 See [Work execution audit page](work-execution-audit.md)
+
+## Performance
+
+### Asynchronous connetor execution
```
Comment (educhastenier) [inline md/release-notes.md]: typo
```suggestion
### Asynchronous connector execution
```

File: md/connectors-execution.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Connectors execution
+
+## What are connectors
+
+Connectors are extension point that can be added to a process to extend Bonita Runtime capabilities. They are a piece of code having inputs defined as `Expressions` and outputs that are defined as `Operations`.
```
Comment (educhastenier) [inline md/connectors-execution.md]:
```suggestion
Connectors are extension points that can be added to a process to extend Bonita Runtime capabilities. They are a piece of code having inputs defined as `Expressions` and outputs defined as `Operations`.
```

File: md/connectors-execution.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## Executiion mechanism
```
Comment (educhastenier) [inline md/connectors-execution.md]:
```suggestion
## Execution mechanism
```

File: md/release-notes.md [hunk 4]
```diff
@@ -122,3 +122,10 @@ A work execution audit mechanism has been introduced. It can be activated
...
+
+Connectors are executed in an asynchrous manner. in earlier version each work was waiting for the connector to end before processing other workload resulting in degraded performances if few connectors have a long execution time.
```
Comment (educhastenier) [inline md/release-notes.md]:
```suggestion
Connectors are executed in an asynchrous manner. In earlier versions each work was waiting for the connector to end before processing other workload. This resulted in degraded performance if few connectors had a long execution time.
```

File: md/release-notes.md [hunk 5]
```diff
@@ -122,3 +122,10 @@ A work execution audit mechanism has been introduced. It can be activated
...
+Worker thread are now releases as soon as the execution of the connector is triggered. see [connector execution page](connectors-execution.md) for more details.
```
Comment (educhastenier) [inline md/release-notes.md]:
```suggestion
Worker threads are now released as soon as the execution of the connector is triggered. see [connector execution page](connectors-execution.md) for more details.
```

**Review by educhastenier — CHANGES_REQUESTED** [review]
a few formatting remarks

**Comment by baptistemesta** [comment]
thanks for fixing my messy english :)

**Review by educhastenier — APPROVED** [review]

**Pull request merged**